chapters/remote/remote-pods.md

```diff
@@ -0,0 +1,42 @@
+# Remote Pods with JinaD
+
+### Prerequisites
+
+Before you start, make sure you have read the [prerequisites for using JinaD](https://docs.jina.ai/chapters/remote/jinad.html#prerequisites).
+
+## Inside a Flow
+
+A common case of using Jina remotely is to have a Flow running locally with Pod on the remote. Following code creates a Flow with a remote Pod on `1.2.3.4` and uploads the pod yml to remote using `upload_files`. All remote logs are collected by the Flow.
+
+#### Python
+
+```python
+from jina import Flow, Document
+f = (Flow()
+     .add(uses='_logforward',
+          host='1.2.3.4:8000',
```

Comment:
again for being explicit I would mention what all the variables are here.

- 1.2.3.4 (you already mention this above, but maybe make it in the form of a list)
- 8000
- custom_encoder.yml

Reply:
This is mentioned in the pre-requisite.